src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php

```diff
@@ -73,6 +73,25 @@ public function testSendWithDelay()
         $this->assertGreaterThan($now, $available_at);
     }
 
+    public function testSendWithNegativeDelay()
+    {
+        $this->connection->send('{"message": "Hi, I am not actually delayed"}', ['type' => DummyMessage::class], -600000);
+
+        $qb = $this->driverConnection->createQueryBuilder()
+            ->select('m.available_at')
+            ->from('messenger_messages', 'm')
+            ->where('m.body = :body')
+            ->setParameter('body', '{"message": "Hi, I am not actually delayed"}');
+
+        // DBAL 2 compatibility
+        $result = method_exists($qb, 'executeQuery') ? $qb->executeQuery() : $qb->execute();
+
+        $available_at = new \DateTimeImmutable($result->fetchOne());
```

Comment:
```suggestion
        $availableAt = new \DateTimeImmutable($result->fetchOne());
```

Reply:
Thanks for your feedback. I copied this from an existing test. Is it desired to update existing code in this same PR?

Reply:
Yes please update it right away 👍 Thanks

Reply:
Done!